frontend/.env

```
@@ -1,2 +1,7 @@
-REACT_APP_ROOT_URL="https://chapter-website-backend.herokuapp.com"
-REACT_APP_API_URL="http://localhost:8000"
\ No newline at end of file
+VITE_ROOT_URL="https://chapter-website-backend.herokuapp.com"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)